primitives/api/proc-macro/src/decl_runtime_apis.rs

```diff
@@ -586,16 +614,33 @@ impl<'a> ToClientSideDecl<'a> {
 		let block_id = self.block_id;
 		let crate_ = self.crate_;
 
-		Some(parse_quote! {
-			#[doc(hidden)]
-			fn #name(
-				&self,
-				at: &#block_id,
-				context: #crate_::ExecutionContext,
-				params: Option<( #( #param_types ),* )>,
-				params_encoded: Vec<u8>,
-			) -> std::result::Result<#crate_::NativeOrEncoded<#ret_type>, #crate_::ApiError>;
-		})
+		// TODO: fix copy-pasted code here!!!
```

Comment:
I'm not sure this code will stay here so I left the copy-pasted version. Will be fixed.